llvm/include/llvm/Support/JSON.h

```diff
@@ -482,6 +482,9 @@ class Value {
   friend class Object;
 
   template <typename T, typename... U> void create(U &&... V) {
+#ifndef _LIBCPP_HAS_NO_ASAN
+    __asan_unpoison_memory_region(&Union, sizeof(T));
```

Comment:
This one needs comment. This unpoisoning looks very random

Reply:
Added here: https://github.com/llvm/llvm-project/pull/79065